examples/resources/configuration/istio/gateways.yaml

```diff
@@ -0,0 +1,212 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: gateways
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: gateways
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  dataSources: |-
+    {
+      "podSelector": {
+        "resource": {
+          "kind": "Pod",
+          "version": "v1",
+        },
+        "filter": "$matchByLabelSelector($item, $root.spec.selector)"
+      }
+    }
+  details: |
+    {
+      "header": [
+        {
+          "source": "spec.selector",
+          "widget": "Labels",
+          "name": "spec.selector"
+        }
+      ],
+      "body": [     
+        {
+          "widget": "Table",
+          "source": "spec.servers",
+          "name": "spec.servers",
+          "children": [
+            { "source": "$item.port.name", "name": "spec.servers.port.name" },
+            {
+              "widget": "JoinedArray",
+              "separator": 'break',
+              "source": "$item.hosts", 
+              "name": "spec.servers.hosts" 
+            },
+            { "source": "$item.port.number", "name": "spec.servers.port.number" },
+            { "source": "$item.tls.mode", "name": "spec.servers.tls.mode" },
+            {
+              "widget": "ResourceLink", 
+              "source": "$item.tls.credentialName", 
+              "name": "spec.servers.tls.credentialName",
+              "resource": {
+                "name": "data",
+                "namespace": "istio-system",
+                "kind": "'Secret'"
+              }
+            }
+          ]
+        },
+        {
+          "widget": "Panel",
+          "name": "spec.selector",
+          "children": [
+            { 
+              "source": "$podSelector()",
+              "widget": "ResourceList",
+              "disableCreate": true,
+              "isCompact": true
+            }
+          ],
+          "header": [
+            {
+              "source": "spec.selector",
+              "widget": "Labels",
+              "name": "spec.selector",
+              "visibility": "spec.selector"
```

Comment:
It won't be visible unless I remove the `"visibility": "spec.selector"`. Looks like jsonata doesn't quite get it, maybe try [boolean](https://docs.jsonata.org/boolean-functions#boolean) or [exists](https://docs.jsonata.org/boolean-functions#exists)?